andela/hc-silicon
# #157694757 send alerts via sms

#### What does this PR do?
- Add a section on the dashboard for the user to set their preferred alert mode
- Sends SMS alerts when the alert mode is set to phone

#### Description of Task to be completed?
When a user sets the alert mode to phone and a job run too often or goes down he/she receive alerts via phone SMS on the phone number provided
#### How should this be manually tested?
- Run the application and click the check link to create a new check
- Go to account settings and set the alert mode to phone and provide a phone number to receive alerts.
- Run the job too often or let it go down and you will receive alerts through SMS instead of default email method.
- When running locally make sure send alerts is running by running:-
`./manage.py runserver & ./manage.py sendalerts`
#### What are the relevant pivotal tracker stories?
[Send alerts via multiple methods](https://www.pivotaltracker.com/story/show/157694757)

Screenshots
A sample of the dashboard where the user will change alert mode
![image](https://user-images.githubusercontent.com/22454909/41536656-ea37e2d0-730e-11e8-9afb-7a8675c66ab2.png)

![image](https://user-images.githubusercontent.com/22454909/41511781-523edaac-7286-11e8-87ad-ceba13b2773c.png)

A sample of SMS alerts sent
![image](https://user-images.githubusercontent.com/22454909/41534864-cbd9b36e-7308-11e8-890c-9b69687f6641.png)



@Awesome94  @Felistas  @daktari01 @kzyGit  @tonykimathi  @muhozi
